src/Package.appxmanifest

```diff
@@ -25,7 +25,7 @@
       </OutOfProcessServer>
     </Extension>
   </Extensions>
-  <Identity Name="Microsoft.Windows.DevHome.Dev" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="0.1299.449.658" />
+  <Identity Name="Microsoft.Windows.DevHome.Dev" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="0.0.0.0" />

```

Comment:
Was this one updated by mistake? Or is 0.0.0.0 all good?

Reply:
Confirmed with @EricJohnson327 that the checked in manifest version should be 0.0.0.0, so this PR is correct and is inadvertently fixing another error.